Content.Server/CrewManifest/CrewManifestSystem.cs

```diff
@@ -135,6 +137,23 @@ private void OpenEuiFromBui(EntityUid uid, CrewManifestViewerComponent component
         OpenEui(owningStation.Value, sessionCast, uid);
     }
 
+    private void OnViewManifestAction(EntityUid uid, CrewManifestViewerComponent component, ViewManifestActionEvent msg)
+    {
+        _entityManager.TryGetComponent(uid, out ActorComponent? actor);
+        var owningStation = _stationSystem.GetOwningStation(uid);
+        if (owningStation == null || actor == null || actor.PlayerSession is not IPlayerSession sessionCast)
```

Comment:
You shouldn't copy-paste code between methods like this as any bugfixes need to be copy-pasted assuming whoever fixes them sees them both. They should both be calling the same method.